### PR TITLE
Sample update 2018

### PR DESCRIPTION
**Pull request by kalyankrishna1**

### CODE REVIEW

File: AppCreationScripts/createdApps.html [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<html><body><table>
```
Comment (jmprieur) [inline AppCreationScripts/createdApps.html]: You don't want to commit createdApps.html as this is generated by the creation scripts.

File: AppCreationScripts/Configure.ps1 [hunk 2]
```diff
@@ -199,7 +199,7 @@ Function ConfigureApplications
    # Add Required Resources Access (from 'service' to 'Microsoft Graph')
    Write-Host "Getting access from 'service' to 'Microsoft Graph'"
    $requiredPermissions = GetRequiredPermissions -applicationDisplayName "Microsoft Graph" `
-                                                -requiredDelegatedPermissions "User.Read";
+                                                -requiredDelegatedPermissions "User.Read" `
```
Comment (jmprieur) [inline AppCreationScripts/Configure.ps1:202]: I don't think we need the trailing ` here as there isn't a next line?

File: OAuth2-UserIdentity/Web.config [hunk 3]
```diff
@@ -16,8 +16,8 @@
     <add key="webpages:Enabled" value="false" />
     <add key="ClientValidationEnabled" value="true" />
     <add key="UnobtrusiveJavaScriptEnabled" value="true" />
-    <add key="ida:ClientId" value="f8e269ae-8eb2-4c73-8d04-7640f510fda1" />
-    <add key="ida:AppKey" value="sypghJlPLg3lZHW6HRXoFEUo9ADFwbVBwyRYDa9PxDc=" />
+    <add key="ida:ClientId" value="[Enter client ID as obtained from Azure Portal, e.g. 82692da5-a86f-44c9-9d53-2f88d52b478b]" />
+    <add key="ida:AppKey" value="[Enter app key as obtained from Azure Portal, e.g. rZJJ9bHSi/cYnYwmQFxLYDn/6EfnrnIfKoNzv9NKgbo=]" />
     <add key="ida:Tenant" value="kalyankrishna.in" />
```
Comment (jmprieur) [inline OAuth2-UserIdentity/Web.config]: Do you keep the tenant to your own tenant ?

**Review by jmprieur — COMMENTED** [review]
A few things to remove?

**Comment by jmprieur** [comment]
Got Could not find a part of the path 'C:\ghs\s\active-directory-dotnet-webapp-webapi-oauth2-useridentity\OAuth2-UserIdentity\bin\roslyn\csc.exe'. while testing

File: OAuth2-UserIdentity/Web.config (hunk 3 above)
Comment (kalyankrishna1) [reply OAuth2-UserIdentity/Web.config]: This was an oversight, fixed

File: AppCreationScripts/Configure.ps1 (hunk 2 above)
Comment (kalyankrishna1) [reply AppCreationScripts/Configure.ps1:202]: It gives a syntax error to me, I've also provided a PR for the sample generator for this. Please check this for me. Thanks

File: AppCreationScripts/createdApps.html (hunk 1 above)
Comment (kalyankrishna1) [reply AppCreationScripts/createdApps.html]: Removed a bunch of files. It was a mistake.

**Pull request merged**